pkg/asset/manifests/infrastructure.go

```diff
@@ -213,6 +219,10 @@ func (i *Infrastructure) Generate(dependencies asset.Parents) error {
 				IngressIPs:           installConfig.Config.VSphere.IngressVIPs,
 			}
 		}
+		if installConfig.Config.VSphere.LoadBalancer != nil && installConfig.Config.VSphere.LoadBalancer.Type != "" {
```

Comment:
We should set a default value for `installConfig.Config.VSphere.LoadBalancer.Type` to  `OpenShiftManagedDefault` in [SetPlatformDefaults()](https://github.com/openshift/installer/blob/master/pkg/types/vsphere/defaults/platform.go#L9) (or [installconfig.go](https://github.com/openshift/installer/blob/master/pkg/types/defaults/installconfig.go) if you want to do it globally for all on-prem platforms) so that this value is never empty, right? Or is there a reason to check for empty values?

Reply:
Like I explained later, this field doesn't exist on stable CRD (see my API PR), so I don't think we can set a default here usable by both stable & TechPreview deployments. I think we should leave it that way for now.

Maybe we can confirm with @patrickdillon whether it's good practice or not.

Reply:
Oh good point, it didn't occur to me that because it was tech preview we didn't want to set platform status for clusters unless they enabled the tech preview. That said, allowing empty values is confusing as hell - can we use a default _when_ the user enables tech preview?

Reply:
The default is at the API level, I'm not sure we want/need a default here.